packages/foreman/rubygem-sinatra/rubygem-sinatra.spec

```diff
@@ -106,10 +103,14 @@ cp -a .%{gem_dir}/* \
 %doc %{gem_instdir}/README.ru.md
 %doc %{gem_instdir}/README.zh.md
 %{gem_instdir}/Rakefile
+%doc %{gem_instdir}/SECURITY.md
 %{gem_instdir}/examples
 %{gem_instdir}/sinatra.gemspec
 
 %changelog
+* Fri May 13 2022 Eric D. Helms <ericdhelms@gmail.com> - 1:2.2.0-1
```

Comment:
changelog_epoch_version_release: 1:2.2.0-1
msg: changelog entry missing for 2.2.0-1
specfile_epoch_version_release: 2.2.0-1

changelogs disagreeing with epoch

Reply:
Thanks for the catch. Updated.